app/assets/javascripts/site.js

```diff
@@ -237,14 +237,17 @@ var Downloads = {
 
   getOSFilter: function() {
     var os = location.href.substring(location.href.lastIndexOf("/") + 1);
```

Comment:
i feel this is really hacky, but didn't want to introduce an huge change for this small task. I may tackle this in the (near) future